extension-config.json

```diff
@@ -0,0 +1,45 @@
+{
+  "version": "1.0.0",
+  "id": "@shopgate/ext-terms-and-conditions",
+  "components": [
+    {
+      "id": "TermsAndConditions",
+      "path": "frontend/portals/PaymentBarBefore",
```

Comment:
I failed to make this extension run. I did some small investigation and this path doesn't exist.
Please clean up the code and make sure it's ready for a review.

Reply:
Ah my stupid mistake. I changed this before leaving yesterday.
Fixed.